client/blocks/conversation-caterpillar/docs/example.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { ConversationCaterpillar } from 'blocks/conversation-caterpillar';
+import { posts } from 'blocks/reader-post-card/docs/fixtures';
+import { commentsTree } from 'blocks/conversations/docs/fixtures';
+
+const ConversationCaterpillarExample = () => {
+	return (
+		<div className="design-assets__group">
+			<ConversationCaterpillar commentsTree={ commentsTree } blogId={ 123 } postId={ 12 } />
```

Comment:
good question. what are our options?
1. caterpillar needs to be a subcomponent of comment-list
2. lift state to a new component that houses both caterpillars + comment lists
3. include the info in redux